class UserNotFoundException(Exception):
    """
    Exception for user not found.
    """

    pass


class UserAlreadyExistsException(Exception):
    """
    Exception for user already exists.
    """

    pass
